docs/modules/databases/influxdb.md

```diff
@@ -1,8 +1,75 @@
 # InfluxDB Module
 
-Testcontainers module for InfluxData [InfluxDB](https://github.com/influxdata/influxdb).
+Testcontainers module for InfluxData [InfluxDB](https://www.influxdata.com/products/influxdb/).
 
-## Usage example
+## Important Note
+
+They are breaking changes in InfluxDB v2.x. For more information refer to the
+main [documentation](https://docs.influxdata.com/influxdb/v2.0/upgrade/v1-to-v2/). InfluxDB
+official [container registry](https://hub.docker.com/_/influxdb) on docker hub.
+
+## InfluxDB V2.x Usage example
+
+Running influxDbContainer as a stand-in for InfluxDB in a test:
+
+```java
+public class SomeTest {
+
+    @ClassRule
+    public static final InfluxDBV2Container influxDbContainer =
+        new InfluxDBContainerV2<>(DockerImageName.parse("influxdb"));
+
+    @Test
+    public void someTestMethod() {
+        final InfluxDBClient influxDB = influxDbContainer.getInfluxDBClient();
+        // Rest of the test
+    }
+}
+```
+
+This will create a InfluxDB container with the latest tag (
+available [tags](https://hub.docker.com/_/influxdb?tab=tags&page=1&ordering=last_updated)). The influxDB will be setup
```

Comment:
Please could we avoid encouraging use of `latest` tags? It results in non-repeatable builds, so is something we prefer to steer people away from.

Also, please could we use `codeinclude` macros for these code snippets, so that they are verified compilable/working?

The [Elasticsearch module docs](/blob/master/docs/modules/elasticsearch.md) are a good example of both of these practices.